src/store/traction/saphyr/requests/actions.js

```diff
@@ -11,10 +12,44 @@ const setRequests = async ({ commit, getters }) => {
   }
 }
 
+const exportSampleExtractionTubesIntoTraction = async ({ getters }, tubes) => {
+  let body = {
```

Comment:
It probably makes sense to change the let assignments to a const where they are immutable.